src/index.js

```diff
@@ -0,0 +1,43 @@
+import './main.css';
+
+const divEl = document.querySelector('.task-ul');
+
+const todos = [
+  {
+    description: 'Creating a frontend project',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Baking cakes',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Reading about local storage',
+    completed: false,
+    index: 3,
+  },
+  {
+    description: 'Cleaning the house',
+    completed: false,
+    index: 4,
+  },
+];
```

Comment:
- [ ] **Required:** You are required to render the list in the order of the indexes. Kindly fix it.

![image](https://user-images.githubusercontent.com/27709832/171253319-67d30ce1-5e3d-4f3e-ad3f-2df888e53f61.png)

**_Hint_** You could use `Array.sort((a,b) => a.index - b.index)`